parts/k8s/containeraddons/kubernetesmasteraddons-cluster-autoscaler-deployment.yaml

```diff
@@ -78,7 +78,7 @@ metadata:
     k8s-addon: cluster-autoscaler.addons.k8s.io
     k8s-app: cluster-autoscaler
     kubernetes.io/cluster-service: "true"
-    addonmanager.kubernetes.io/mode: EnsureExists
+    addonmanager.kubernetes.io/mode: "EnsureExists"
```

Comment:
not sure if we want to revert these since it's not breaking

Reply:
Good point--I was going by the bug report diff, but AFAICT it's legit not to quote these string values so perhaps we should leave them be.

Reply:
Erring on the side of caution here. The unquoting of this was done to remove superfluous characters in an effort to economize on customScript char count. Given the new addons refactor change, I thought it's ok to change it back. Thoughts @mboersma @CecileRobertMichon

Reply:
Certainly open to reverting as suggested by @CecileRobertMichon